lib/rest/api_client.cpp

```diff
@@ -183,6 +197,15 @@ namespace disccord
                     return pipeline->propagate(req);
                 });
             }
+            
+            void rest_api_client::set_content_length(uint64_t length)
```

Comment:
There is no need for this method to exist: it is handled automatically by cpprestsdk's http_request type.